include/quicr/subscribe_track_handler.h

```diff
@@ -225,6 +225,12 @@ namespace quicr {
          */
         void SupportNewGroupRequest(bool is_supported) noexcept;
 
+        /**
+         * @brief Indicate if subscribe handler can send new group requests or not
+         * @return True if new group requests are supported.
+         */
+        bool NewGroupRequestSupported() const noexcept { return support_new_group_request_; }
```

Comment:
Very close naming to `SupportNewGroupRequest()`.  Maybe rename this to `IsNewGroupRequestSupported()` 

This also could rathole with MOQT adding more params that could cause us to have  `SetXSupported()` and `IsXSupported()` pattern for various features.